src/Attribute/StringAttribute.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * This file is part of the php-state project.
+ *
+ * (c) Yannick Voyer <star.yvoyer@gmail.com> (http://github.com/yvoyer)
+ */
+
+namespace Star\Component\State\Attribute;
+
+use Webmozart\Assert\Assert;
+
+final class StringAttribute implements StateAttribute
```

Comment:
Implement Null attribute that always match value as long as attribute is found on state.